include/llvm/IR/DTUpdater.h

```diff
@@ -0,0 +1,72 @@
+//===- DTUpdater.h - DomTree/Post DomTree Updater ---------------*- C++ -*-===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the DTUpdater class, which provides a uniform way to
+// update dominator related data structures.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_DTUPDATER_H
+#define LLVM_DTUPDATER_H
+
+#include "llvm/Analysis/PostDominators.h"
+#include "llvm/IR/Dominators.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/Support/GenericDomTree.h"
+
+namespace llvm{
```

Comment:
You can use clang-format to format your code. It uses the llvm coding style rules by default, so it's enough to run it without any special flags. There's also a script that formats diffs, which will become useful once you start sending patches for upstream review.